whois-update/src/main/java/net/ripe/db/whois/update/domain/UpdateMessages.java

```diff
@@ -629,6 +629,10 @@ public static Message creatingRipeMaintainerForbidden() {
         return new Message(Type.ERROR, "You cannot create a RIPE NCC maintainer");
     }
 
+    public static Message updatingRipeMaintainerSSOForbidden() {
+        return new Message(Type.ERROR, "You cannot update sso attributes as maintainer is synchronized in LIR portal");
```

Comment:
re-word text as 'You cannot update SSO auth attribute(s), because the maintainer is synchronised from the LIR Portal'.